thirdparty/README.md

```diff
@@ -1,5 +1,12 @@
 # Third party libraries
 
+## assimp
+
+- Upstream: https://github.com/assimp
```

Comment:
Should be http://github.com/assimp/assimp

Reply:
Done.